task2/tests/fibonacciTest.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+require_once __DIR__ . '/../fibonacci.php';
+
+class fibonacciTest extends \PHPUnit\Framework\TestCase
+{
+    public function test_digit()
```

Comment:
Add one more test. Find the 100 digit number in google and compare it with your method call.